payjoin/src/receive/multiparty/mod.rs

```diff
@@ -248,3 +248,68 @@ impl FinalizedProposal {
 
     pub fn v2(&self) -> &[v2::UncheckedProposal] { &self.v2_proposals }
 }
+
+#[cfg(test)]
+mod test {
+
+    use std::any::{Any, TypeId};
+
+    use payjoin_test_utils::{BoxError, PARSED_ORIGINAL_PSBT};
```

Comment:
Doesn't seem like a big reason to import and use `BoxError` or results if the functions return `Ok(())` and the `?` isn't used anywhere.